versioned_docs/version-0.13/installation/index.mdx

```diff
@@ -29,12 +30,12 @@ Botkube has two components that need to be installed.
 
 <IntegrationList
   integrations={[
-    { name: "Slack", image: "/images/integrations/slack.svg", link: "slack" },
-    { name: "Mattermost", image: "/images/integrations/mattermost.svg", link: "mattermost" },
-    { name: "Discord", image: "/images/integrations/discord.svg", link: "discord" },
-    { name: "Teams", image: "/images/integrations/msteams.svg", link: "teams" },
-    { name: "ElasticSearch", image: "/images/integrations/elasticsearch.svg", link: "elasticsearch" },
-    { name: "Webhook", image: "/images/integrations/webhooks.svg", link: "webhook" },
+    { name: "Slack", image: "/images/integrations/slack.svg", link: "installation/slack" },
```

Comment:
The links don't work for current versions. For 0.13 and 0.14 it is fine. Please fix it - thanks!